pulsar-zookeeper-utils/src/test/java/org/apache/pulsar/zookeeper/ZkIsolatedBookieEnsemblePlacementPolicyTest.java

```diff
@@ -161,21 +161,24 @@ public void testBasic() throws Exception {
         }
 
         try {
-            isolationPolicy.replaceBookie(3, 3, 3, Collections.emptyMap(), new HashSet<>(ensemble),
+            isolationPolicy.replaceBookie(3, 3, 3, Collections.emptyMap(), ensemble,
                     new BookieSocketAddress(BOOKIE5), new HashSet<>());
             fail("should not pass");
         } catch (BKNotEnoughBookiesException e) {
             // ok
         }
-
+        /*
```

Comment:
Fix this. This test was failing, but it's more to do with the test relying on implicit behaviour.